### PR TITLE
Convert Completed from PushPullStage to GraphStage 

### PR DESCRIPTION
**Pull request by pchordia15**
Issue #20288

### CODE REVIEW

**Comment by pchordia15** [comment]
#20288

**Comment by drewhk** [comment]
OK TO TEST

**Comment by ktoso** [comment]
LGTM, thanks!
Not much else to be discussed here I think so merging

File: akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala [hunk 1]
```diff
@@ -568,10 +568,23 @@ private[akka] final case class Buffer[T](size: Int, overflowStrategy: OverflowSt
 /**
  * INTERNAL API
  */
-private[akka] final case class Completed[T]() extends PushPullStage[T, T] {
-  override def onPush(elem: T, ctx: Context[T]): SyncDirective = ctx.finish()
+private[akka] final case class Completed[T]() extends GraphStage[FlowShape[T, T]] {
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala]: it could be made an object but this PR is good as is

**Comment by pchordia15** [comment]
Thanks @ktoso

**Comment by ktoso** [comment]
Thank _you_ :-)

**Pull request merged**